docs/java/how-to-get-the-result-of-an-activity-execution-in-java.md

```diff
@@ -0,0 +1,54 @@
+---
+id: how-to-get-the-result-of-an-activity-execution-in-java
+title: How to get the result of an Activity Execution
+sidebar_label: Activity Execution Result
+description: To get the results of an asynchronously invoked Activity method, use the `Promise` `get` method to block until the Activity method result is available.
+tags:
+  - java
+  - developer-guide
+---
+
+To get the results of an asynchronously invoked Activity method, use the `Promise` `get` method to block until the Activity method result is available.
+
+Sometimes an Activity Execution lifecycle goes beyond a synchronous method invocation.
+For example, a request can be put in a queue and later a reply comes and is picked up by a different Worker process.
+The whole request-reply interaction can be modeled as a single Activity.
+
+To indicate that an Activity should not be completed upon its method return, call `ActivityExecutionContext.doNotCompleteOnReturn()` from the original Activity thread.
```

Comment:
I think this is info that we build to?
Seems out of place at the beginning of this.